package.json

```diff
@@ -79,6 +79,7 @@
     "http-server": "^14.1.1",
     "https-browserify": "^1.0.0",
     "mini-css-extract-plugin": "^2.2.2",
+    "near-bos-webcomponent": "^0.0.2",
```

Comment:
can you remove this dependency? it should not depend on "itself".

Reply:
Lol yeah, my bad.